pkgs/os-specific/linux/kernel/perf.nix

```diff
@@ -15,6 +15,13 @@ stdenv.mkDerivation {
 
   inherit (kernel) src;
 
```

Comment:
```suggestion

assert !(versionAtLeast kernel.version "5.19") -> throw "perf with binutils >= 2.39 requires kernel >= 5.19";

Reply:
I don't understand this assert. It does not mention binutils version in the constraints. Where does that requirement come from?

Also note that `master` branch does not yet have `binutils-2.39`.

Reply:
> I don't understand this assert. It does not mention binutils version in the constraints. Where does that requirement come from?

I am suggesting that you include this change in your PR #185297, which bumps the binutils version to 2.39.

Github won't let me ```suggest this change in that PR.  Apparently I can only ```suggest changes within three lines of something the PR already touches.

Reply:
Ahh, that makes sense.